DependencyInjection/Configuration.php

```diff
@@ -724,6 +724,7 @@ private function getOrmCacheDriverNode(string $name): ArrayNodeDefinition
                 ->scalarNode('type')->defaultNull()->end()
                 ->scalarNode('id')->end()
                 ->scalarNode('pool')->end()
+                ->booleanNode('is_psr6')->defaultFalse()->end()
```

Comment:
@nicolas-grekas didn't like it last time in https://github.com/doctrine/DoctrineBundle/pull/1332#pullrequestreview-658751708. @nicolas-grekas are you ok with it now?

This would probably be possible to solve via compiler pass instead, but it would require more effort.

Reply:
Indeed via compiler pass it should be working. 

I'm currently looking into a related fix here https://github.com/dmaicher/doctrine-test-bundle/issues/158

where I also need to know which case it is: doctrine/cache or psr6

Currently testing with an approach like this inside a compiler pass

```php
        if (is_subclass_of($originalCacheServiceDefinition->getClass(), CacheItemPoolInterface::class)) {
            $cache->setClass(Psr6StaticArrayCache::class);
            // ...
        } elseif (is_subclass_of($originalCacheServiceDefinition->getClass(), Cache::class)) {
            $cache->setClass(StaticArrayCache::class);
            // ...
        }
```